### ISSUE TITLE
Get directional changes when move away from each other when farthest apart

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#164]
This bug was discovered while I was reviewing https://github.com/phetsims/ratio-and-proportion/issues/152.

Steps to reproduce in a11y view:
1. move left hand to 0
2. with keyboard navigation get to the the max with right hand. alert should be "Farthest from left hand, far from challenge ratio." or something.
3. press up arrow (no change in position because it is already at max), now I get an alert that says: "Closer to left hand, far from challenge ratio."
4. press up arrow again (no change in position because it is already at max), not I get an alert that says: "Farthest from left hand, far from challenge ratio."

(3) and (4) should be calculated as the exact same alert as (2), and so should not alert (no duplicates allowed currently). I'll fix this.

**Comment by zepumph** [comment]
Fixed above. closing
